lib/swagger-oauth.js

```diff
@@ -165,6 +197,28 @@ function handleLogin() {
   return;
 }
 
+function handlePasswordFlow(auth) {
+  var tokenUri = auth.tokenUrl;
+
+  var authParams = {
+    grant_type: 'password',
+    client_id: encodeURIComponent(clientId),
```

Comment:
may I suggest adding the client_secret here as well?
client_secret: encodeURIComponent(clientSecret),

at least it helped me using password flow with my REST API endpoint which requird a secret as well for token generation